setup.py

```diff
@@ -87,7 +87,7 @@ def get_version(version_file):
     package_data={
         "pe_mailer": ["data/*"],
         "pe_reports": ["data/shell/*.pptx", "data/*.ini"],
-        "pe_source": ["data/*"],
+        "pe_source": ["data/*", "data/shodan/*", "data/sixgill/*", "data/pe_db/*"],
```

Comment:
@cduhn17 All the config files are in data/pe_db/*. I separated the shodan and sixgill folders because they contain the specific API requests. Also, future sources such as DNS Monitor and DNS Twist require worker files (e.g. tld list) which I think will be good in their own folders.

Reply:
Suggessed changes will be addressed in another PR. Issue link: https://github.com/cisagov/pe-reports/issues/152

Reply:
@aloftus23 
Per our conversation lets leave the "data" directories in place as they are and create an issue to refactor all the code that interacts with the database that is in the "pe_reports/data" and move it over into "pe_sources/data" for a one stop location of dealing with data or database connections.